gui.h

```diff
@@ -855,6 +886,111 @@ color_t color_blend(color_t src, color_t dst)
 	};
 }
 
+void rgb_to_hsv(r32 r, r32 g, r32 b, r32 *h, r32 *s, r32 *v)
+{
+	const r32 rgb_min = min(r, min(g, b));
+	const r32 rgb_max = max(r, max(g, b));
```

Comment:
min/max are macros; they're identical to fmin & fmax.  The reason both exists is in case you include 'fmath.h' and don't want 'imath.h'.  I don't think that's a great reason anymore, but I made that call maybe 2 years ago.  Probably should just put min/max into core.h, but for now I'll change to fminf/fmaxf, since fmin/fmax are in 'dmath.h'.